src/flyte/_cache/local_cache.py

```diff
@@ -0,0 +1,121 @@
+from pathlib import Path
+
+from flyteidl.core import interface_pb2
+from sqlalchemy import Column, LargeBinary, String, delete
+from sqlalchemy.ext.asyncio import AsyncEngine, AsyncSession, async_sessionmaker, create_async_engine
+from sqlalchemy.orm import DeclarativeBase
+
+from flyte._internal.runtime import convert
+from flyte._protos.workflow import run_definition_pb2
+
+CACHE_LOCATION = "~/.flyte/local-cache"
+
+
+class Base(DeclarativeBase):
+    pass
+
+
+class CachedOutput(Base):
```

Comment:
Could we do `CachedOutput(DeclarativeBase)` instead? Any reason to create another base?